base/src/base_layers.ts

```diff
@@ -1142,6 +1169,23 @@ export function labels_layers(source: string, c: Theme): any[] {
         "text-halo-width": 1.5,
       },
     },
+    {
+      id: "pois",
```

Comment:
This shows text for POIs at zoom 17+:

<img width="803" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/19c5690c-c63e-49d7-a67e-eb1e673a65e3">